driver-core/src/test/resources/client-side-encryption/legacy/timeoutMS.json

```diff
@@ -110,7 +110,7 @@
             "listCollections"
           ],
           "blockConnection": true,
-          "blockTimeMS": 60
```

Comment:
The timeouts are increased to accommodate for the additional time required for SSL handshake when authentication is enabled. This adjustment ensures that the timeout does not expire before the expected steps in our tests are completed.

Reply:
Great suggestion, thank you! I've added the comment.